libp2p/peerinfo.nim

```diff
@@ -9,14 +9,19 @@
 
 {.push raises: [Defect].}
 
-import std/[options, sequtils, hashes]
-import pkg/[chronos, chronicles, stew/results]
-import peerid, multiaddress, crypto/crypto, errors
+import std/[options, sequtils, hashes, times]
+import pkg/[chronos, chronicles, stew/byteutils, stew/results]
+import peerid, multiaddress, crypto/crypto, signed_envelope, routing_record, errors
 
-export peerid, multiaddress, crypto, errors, results
+export peerid, multiaddress, crypto, signed_envelope, errors, results
 
 ## Our local peer info
 
+## Constants relating to signed peer records
+const
+  EnvelopePayloadType* = "/libp2p/routing-state-record".toBytes() # payload_type for routing records as spec'ed in RFC0003
+  EnvelopeDomain* = "libp2p-routing-record" # envelope domain as per RFC0002
```

Comment:
Actually, since the spec was done, it has been added to multicodec

`ibp2p-peer-record`
https://github.com/multiformats/multicodec/blob/master/table.csv

We should update our local list, and use this instead

Reply:
Made `EnvelopeDomain` a multicodec in 0d5dfbb and added `libp2p-peer-record` to our list. Afaict we currently only use it in its stringified format.